easybuild/easyconfigs/i/ImageMagick/ImageMagick-7.0.8-11-GCCcore-7.3.0-Ghostscript-9.23-cairo-1.14.12.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'ImageMagick'
+version = '7.0.8-11'
+ghostversion = '9.23'
+cairoversion = '1.14.12'
+versionsuffix = '-Ghostscript-%s-cairo-%s' % (ghostversion, cairoversion)
```

Comment:
@crubb Same here, no need for the `versionsuffix` in this case (since it's the first `ImageMagick` easyconfig using this toolchain).